torchtune/modules/tied_linear.py

```diff
@@ -31,4 +32,12 @@ def __init__(self, tied_module: nn.Module):
             )
 
     def __call__(self, x: torch.tensor) -> torch.tensor:
+        """
+        Args:
+            x (torch.tensor): Input tensor. Should have shape ``(..., in_dim)``, where ``in_dim``
```

Comment:
Why are these typed with lowercase tensor?

Reply:
ugh you caught me. idk and idc, I just wanted these changes to be docstring-only

Reply:
Leaving it as is for now, we can check with @felipemello1 on whether this was by design or not

Reply:
I did it just to see if you guys truly review the PRs, obviously. Congratulations, you passed and test! Wanna update it to upper case?